gemini/evaluation/get_started_with_genai_model_eval_service.ipynb

```diff
@@ -0,0 +1,1744 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    eval_dataset = eval_dataset.remove_columns(["id"])
</pre><p>Is it possible to chain these methods together instead of reassigning to the variable multiple times?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/553/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
<p>done</p>